portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -31,34 +31,37 @@
 
 /** Servlet for handling comments data */
 @WebServlet("/comments")
-public class DataServlet extends HttpServlet {
+public class CommentServlet extends HttpServlet {
 
-  /* Datastore data is represented by entities which have a kind and certain properties,
-   * and the constants below define the kind and property names for comment entities.
-   * Changing them would mean comments saved with a previous definition would not be displayed anymore. */
-  private static final String COMMENT_KIND = "Comment";
-  private static final String CONTENT_PROPERTY = "content";
-
-  /* This is the name of the input form which is defined in the html of the main page */
+  /* The name of the input form which is defined in the html of the main page. */
   private static final String INPUT_FORM = "comment-input";
 
-  /* This specifies what URL the client is redirected to after a POST request. */
+  /* Specifies what URL the client is redirected to after a POST request. */
   private static final String REDIRECT_URL = "/index.html";
 
+  /* The query string parameter for the number of comments to be displayed as defined in the script. */
+  private static final String COMMENT_NUMBER_PARAM = "number";
+
   /**
    * Loads and returns comments from the datastore database.
    */
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
I've tried adding comments and saw that order of the comments seems to be random. Is there a way to sort them by when they were added or is it not supported / should be implemented separately?

Reply:
It is not supported yet and sorting them by timestamp for example would be one of the features I am going to implement next.

Reply:
Sounds great! :)